server/src/main/java/org/elasticsearch/bootstrap/plugins/LoggerTerminal.java

```diff
@@ -48,32 +47,13 @@ public char[] readSecret(String prompt) {
     }
 
     @Override
-    public PrintWriter getWriter() {
-        throw new UnsupportedOperationException();
-    }
-
-    @Override
-    public OutputStream getOutputStream() {
-        throw new UnsupportedOperationException();
-    }
-
-    @Override
-    public PrintWriter getErrorWriter() {
-        throw new UnsupportedOperationException();
-    }
-
-    @Override
-    protected void print(Verbosity verbosity, String msg, boolean isError) {
+    protected void print(Verbosity verbosity, PrintWriter writer, CharSequence msg, boolean newline) {
+        boolean isError = writer == getErrorWriter();
```

Comment:
If this class is initialized with `null` for the outWriter and for the errWriter, won't `isError` always evaluate to `true` here?

Reply:
It would, but passing null for the readers is just a hack LoggingTerminal uses. The expectation is passing null for those means overriding this method.